Samurai/Components/ApiManager.cs

```diff
@@ -21,7 +21,7 @@ public static class ApiManager
 
         public static async Task<Dictionary<string, string>> GetTickerAsDictionaryAsync(string id, string convert = null)
         {
-            using (var request = await HttpClient.GetAsync($"https://api.coinmarketcap.com/v1/ticker/{id}/{(string.IsNullOrEmpty(convert) ? "" : $"?convert={convert}")}").ConfigureAwait(false))
+            using (var request = await Program.HttpClient.GetAsync($"https://api.coinmarketcap.com/v1/ticker/{id}/{(string.IsNullOrEmpty(convert) ? "" : $"?convert={convert}")}").ConfigureAwait(false))
```

Comment:
It seems to be verbose expressions. If you are feeling risks about it, please make unsafe member names safety.